examples/data_masking/sam/template.yaml

```diff
@@ -0,0 +1,57 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Powertools for AWS Lambda (Python) data masking example
+
+Globals: # https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-template-anatomy-globals.html
+  Function:
+    Timeout: 5
+    Runtime: python3.10
+    Tracing: Active
+Resources:
+  MyKMSKey:
+    Type: AWS::KMS::Key
+    Properties:
+      Enabled: true
+      KeyPolicy:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Action: kms:*
+          Resource: "*"
+          Principal:
+            AWS: !Join [ "", [ "arn:aws:iam::", !Ref "AWS::AccountId", ":root" ] ]
+  DataMaskingFunctionExample:
+    Type: AWS::Serverless::Function     # More info about Function Resource: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html
+    Properties:
+      Handler: data_masking_function_example.lambda_handler
+      CodeUri: ../src
+      Description: Data Masking Function Example
+      MemorySize: 1024                  # TODO: Recommended to use 1024 MB due to 
```

Comment:
Is there we want to make the recommendation of what memory size to use based on our load test results? Should we link the load test numbers or the traces or something?

Reply:
Hi @seshubaws! We normally configure 128Mb of memory to prevent the client from copying and pasting our code and incurring some unexpected costs. If we don't have any problems running with 128 MB, it's ok to set 128.

Reply:
Ok, I'll change it to 128, but where should we recommend they use 1024MB since that was the optimal size for latency? I believe @heitorlessa wanted to link to our load testing numbers as well?

Reply:
As per offline conversation with @heitorlessa set to 1024 in the SAM template and added a Note pointing that out and explaining why.